src/Doctrine/ORM/MailchimpExportRepositoryInterface.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Setono\SyliusMailchimpPlugin\Repository;
+namespace Setono\SyliusMailchimpPlugin\Doctrine\ORM;
```

Comment:
Don't you think it is wrong to have the repository interfaces in `Doctrine\ORM` when in fact the interface methods should apply to all storage engines?

Reply:
Maybe. But from other side:
- We have `ResourceLogEntryRepositoryInterface` as an example in original Sylius which placed at `Doctrine/ORM`
- Doctrine MongoDB and PHPCR support will no longer be supported in Sylius 2.0, so probably in future both repositories and interfaces will be placed at `/Repository` directory.